Update to the metric part (#12460)
DataDog/integrations-core: linkerd/README.md

For Linkerd v1, see the [finagle metrics guide][12] for metric descriptions and [this gist][13] for an example of metrics exposed by Linkerd.

> **Note**: Linkerd is a prometheus based integration. Depending on your Linkerd configuration, some metrics might not be exposed by Linkerd. To check which metrics Datadog can collect, you can execute the below curl command in order to output all the exposed metrics by your current configuration. Meaning that if any metric is not present in the curl output, Datadog will not be collecting it.

Reviewer: ```suggestion
**Note**: Linkerd is a Prometheus-based integration. Depending on your Linkerd configuration, some metrics might not be exposed by Linkerd. If any metric is not present in the curl output, the Datadog Agent is unable to collect that particular metric.
```

Simply follow the examples present in the [default configuration][4].

> **Note**: we append, in our code, linkerd metrics ending with '_total' to 'count' (ex. "linkerd.tcp.open_total" will be "linkerd.tcp.open.count")

Reviewer: I don't think this section is necessary. However, I can submit a PR with the updated metric names, since this should have been added back when OpenMetricsV2 support was added for Linkerd. Once that PR is merged, it should be much more clear which metrics are being collected.

Reviewer: https://github.com/DataDog/integrations-core/pull/12664

Author: Removed this part.
Thanks for the PR.
